src/store/RelayPendingQueryTracker.js

```diff
@@ -118,12 +118,14 @@ class PendingFetch {
   _errors: Array<?Error>;
 
   constructor(
+    /* eslint-disable comma-dangle */
```

Comment:
Or perhaps we'll just add the comma there, it's a single comma mishap in a large codebase. Stripping flowtypes might be suboptimal as currently eslint catches unused type imports.